### ISSUE TITLE
can it be used for analysisng videos with two more animals?

### ISSUE THREAD

**Issue by aloksneurobot1** [jbohnslav/deepethogram#59]

**Comment by jbohnslav** [comment]
This is in the revised manuscript which is still in revision. DeepEthogram's major constraint is that it uses frame-level labels; for every single video frame, there can be one more more labels, like `grooming`, `rearing`, or whatever. If there are more than one animal, the label can be something like "interacting", "chasing", or whatever. What it can't do is say: "animal 1: rearing", "animal 2: grooming."

TL;DR: DeepEthogram can do more than one animal, but labels are per-frame, not per-animal.
